Extract manifest generation from WooCommerce docs plugin into a CLI tool (#39161)
woocommerce/woocommerce: tools/monorepo-utils/src/md-docs/README.md

# Markdown Docs CLI tool

This is a CLI tool designed to generate JSON manifests of Markdown files in a directory.

This manifest is currently designed to be consumed by the [WooCommerce Docs](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-docs) plugin.

## Usage

> This command is built on postinstall and can be run from the root of the project.

Reviewer: This sounds like it needs to be run from `/tools/monorepo-utils`

```suggestion
This command is built on postinstall and can be run from monorepo root.
```
